delete_photos.js

```diff
@@ -22,13 +23,18 @@ let buttons = {
 let deleteTask = setInterval(() => {
 
     checkboxes = document.querySelectorAll(ELEMENT_SELECTORS['checkboxClass']);
-
     if (checkboxes.length <= 0) {
+        if (fail == 0) {
+        	console.log("[WARNING] Fail to select checkboxes.");
+        	fail = 1;
+        	return;
+        }
```

Comment:
The approach is hard to understand and more code for the work.
I made another one similar to this one. Could you review to see there is no discrepancies in the behaviour

https://github.com/mrishab/google-photos-delete-tool/pull/10

Reply:
LGTM, i think this PR can be replaced with yours